scripts/install.sh

```diff
@@ -19,8 +19,9 @@ set -ev
 pip install tox
 
 # App Engine tests require the App Engine SDK.
-if [[ "${TOX_ENV}" == "gae" && ! -d ${GAE_PYTHONPATH} ]]; then
-    python scripts/fetch_gae_sdk.py `dirname ${GAE_PYTHONPATH}`
+if [[ "${TOX_ENV}" == "gae" || "${TOX_ENV}" == "cover" ]]; then
+    pip install git+https://github.com/GoogleCloudPlatform/python-repo-tools.git
```

Comment:
1. Why not on PyPI?
1. Does this need to be root or is `pip` on Travis attached to some virtual env?